### ISSUE TITLE
Fix "installing sourmash" on index page

### ISSUE THREAD

**Issue by ctb** [dib-lab/sourmash#669]
On https://sourmash.readthedocs.io/en/latest/, 

```
Installing sourmash
We currently suggest installing the latest pre-release in the sourmash 2.0 series; please see the README file in github.com/dib-lab/sourmash for information. You can use pip or conda equally well.
```

is incorrect.

**Comment by ricky-lim** [comment]
i'm working on this issue

**Comment by luizirber** [comment]
Fixed by #675
